### PR TITLE
#10 Paypal integration

### PR DESCRIPTION
**Pull request by Suseika**
Implemented basic paypal integration.

### CODE REVIEW

File: www/index.html [hunk 1]
```diff
@@ -7,6 +7,8 @@
     <h2>Contacts</h2>
     <p>10007 New York</p>
     <div>© Smartsites 2016</div>
+    <h3>Paypal integration</h3>
+    <div>Click here to paypal</div>
```
Comment (Cookson) [inline www/index.html]: @Suseika Also need a password input

**Comment by Cookson** [comment]
@Suseika Branch has merge conflicts

**Comment by Suseika** [comment]
@Cookson All comments addressed

**Comment by Cookson** [comment]
@Suseika Merging

**Pull request merged**